### DISCUSSION TITLE
Impostare in base alla risposta alla domanda una o più caselle di testo

### DISCUSSION THREAD

**Discussion by cvrboni** [globaleaks/globaleaks-whistleblowing-software#3735] [Q&A]
Volevo chiedere, siccome non riesco a trovarlo nella documentazione, come impostare il fatto che se seleziono, ad esempio, in una domanda l'opzione altro (specificare) il fatto che compaia effettivamente una casella di testo aggiuntiva dove è possibile aggiungere questa opzione, o per fare un altro esempio, se una persona vuole restare anonima o no, se clicca no compaiono delle caselle dove mettere nome e altre informazioni.
Allego foto per fare piu chiaro l'esempio e ringrazio in anticipo
https://imgur.com/a/p271DTf

**Comment by evilaliv3** [comment]
Provate a creare una domanda Altro, di tipo singola linea e poi ad usare la funzione "Attiva domanda" che vi trovate all'interno

**Reply by cvrboni** [reply]
https://imgur.com/a/3t7plGN
ho provato con questa impostazione, però la casellina altro non compare, sto sbagliando qualcosa?
